### ISSUE TITLE
Can write file on AWS S3

### ISSUE THREAD

**Issue by staronline1985** [ironSource/parquetjs#95]
Actually I am using parquetjs in Meteor.js . I want to create a parquet data file . 

ParquetWriter.openFile(schema, filePath) , I am getting below error.

**Comment by a1292717155** [comment]
@staronline1985 
hi, do you find the solution upload parquet to AWS S3

**Comment by staronline1985** [comment]
Yes , I had found the solution to upload parquet to AWS S3 . But I am getting issue with large files. For Example :  I have read  JSON or CSV format file and convert into Parquet format. It keep all  data in memory until unless close to parquet writer. It will not work for me with large file.  My job was read file json file from S3 and convert into parquet format and upload again on S3.

**Comment by staronline1985** [comment]
I think it should be stream based so read data as stream and convert and upload stream to S3

**Comment by a1292717155** [comment]
@staronline1985 I have the same mission. But for now. it just needs me covert local CSV file to parquet and upload s3. But it needs to create a local parquet file and then readFileSync as a buffer to upload . I want to upload S3 directly , don't save local. How to do that?

**Comment by staronline1985** [comment]
> @staronline1985 I have the same mission. But for now. it just needs me covert local CSV file to parquet and upload s3. But it needs to create a local parquet file and then readFileSync as a buffer to upload . I want to upload S3 directly , don't save local. How to do that?

I am also doing same and waiting for parquetjs , if any possibility for same . Otherwise I will go with other repo.

**Comment by ChrisCW1** [comment]
Any updates regarding writing direct to S3?

**Comment by muratcorlu** [comment]
You need to use `ParquetTransformer` as mentioned in #76

**Comment by govthamreddy** [comment]
> You need to use `ParquetTransformer` as mentioned in #76

Do you have an example for doing it?

**Comment by aijazkhan81** [comment]
@govthamreddy do you have a working example of pushing it to s3?

**Comment by magno32** [comment]
https://github.com/ironSource/parquetjs/issues/76#issuecomment-1312158235
